### DISCUSSION TITLE
MacOS Big Sur 11.6.5, Calibre 5.42.0 DeDRM Plugin 7.2.1

### DISCUSSION THREAD

**Discussion by Chandy8626** [apprenticeharper/DeDRM_tools#2066] [General]
Customized the plugin with my Kindle serial number and can't open .azw3 file.  Below is my debug log.  Downloaded file from Amazon.com.  Please help.

calibre Debug log
calibre 5.42  embedded-python: True is64bit: True
macOS-10.16-x86_64-i386-64bit Darwin ('64bit', '')
('Darwin', '20.6.0', 'Darwin Kernel Version 20.6.0: Tue Feb 22 21:10:41 PST 2022; root:xnu-7195.141.26~1/RELEASE_X86_64')
Python 3.8.5
OSX: ('10.16', ('', '', ''), 'x86_64')
Interface language: None
Successfully initialized third party plugins: DeDRM (7, 2, 1)
calibre 5.42  embedded-python: True is64bit: True
macOS-10.16-x86_64-i386-64bit Darwin ('64bit', '')
('Darwin', '20.6.0', 'Darwin Kernel Version 20.6.0: Tue Feb 22 21:10:41 PST 2022; root:xnu-7195.141.26~1/RELEASE_X86_64')
Python 3.8.5
OSX: ('10.16', ('', '', ''), 'x86_64')
Interface language: None
Successfully initialized third party plugins: DeDRM (7, 2, 1)
devicePixelRatio: 1.0
logicalDpi: 72.0 x 72.0
physicalDpi: 136.0000020425151 x 135.0000020274966
Using calibre Qt style: True
[0.00] Starting up...
[0.04] Showing splash screen...
[0.39] splash screen shown
[0.39] Initializing db...
[0.42] db initialized
[0.42] Constructing main UI...
[1.69] main UI initialized...
[1.69] Hiding splash screen
Starting QuickView
DeDRM v7.2.1: Trying to decrypt Techniques of the Selling Writer.azw3
Using Library AlfCrypto Python
Using Library AlfCrypto Python
MobiDeDrm v1.0.
Copyright © 2008-2020 The Dark Reverser, Apprentice Harper et al.
Decrypting Kindle Format 8 ebook: Techniques of the Selling Writer
Found 2 keys to try after 0.1 seconds
Crypto Type is: 2
File is encoded with PID hhu9Yk62CZ.
Decrypting. Please wait . . . [4.18] splash screen hidden
[4.18] Started up in 4.18 seconds with 1 books

Debugging DND event
text/uri-list 77 b'file:///Users/aaron/Downloads/Techniques%20of%20the%20Selling%20Writer.azw3\r\n' 

Firefox rurl: None None
URLS: ['file:///Users/aaron/Downloads/Techniques%20of%20the%20Selling%20Writer.azw3']
Paths: ['/Users/aaron/Downloads/Techniques of the Selling Writer.azw3']
Extensions: frozenset({'azw3'})
. . done
Decryption succeeded after 2.5 seconds
DeDRM v7.2.1: Finished after 2.5 seconds

**Comment by ElleKayEm** [comment]
This log shows successful decryption.
